torch/csrc/jit/passes/python_print.cpp

```diff
@@ -257,8 +257,13 @@ struct PythonPrintPass {
         }
       }
     }
-    if (std::find(deps_table_.cbegin(), deps_table_.cend(), type) ==
-        deps_table_.cend()) {
+    // Need to do actual equality comparison, not a pointer equality.
```

Comment:
Can you make the comment more helpful? I can read that from the code, but I want to know why